Ansible 2.3 can't start tests

Hello. After upgrade to ansible 2.3 starting tests fails with error:
```
============================= test session starts ==============================
platform linux2 -- Python 2.7.12, pytest-3.0.7, py-1.4.33, pluggy-0.4.0
rootdir: /home/enkov/WORK1/playbooks/roles/foo, inifile:
plugins: testinfra-1.5.4
collected 0 items / 1 errorss

==================================== ERRORS ====================================
____________________ ERROR collecting tests/test_default.py ____________________
tests/test_default.py:4: in <module>
    '.molecule/ansible_inventory').get_hosts('all')
/usr/local/lib/python2.7/dist-packages/testinfra/utils/ansible_runner.py:141: in __init__
    self.cli = ansible.cli.CLI(None)
E   TypeError: Can't instantiate abstract class CLI with abstract methods parse, run
============================ pytest-warning summary ============================
WP1 None Module already imported so can not be re-written: testinfra
!!!!!!!!!!!!!!!!!!! Interrupted: 1 errors during collection !!!!!!!!!!!!!!!!!!!!
================== 1 pytest-warnings, 1 error in 0.20 seconds ==================
ERROR: 

  RAN: '/usr/local/bin/testinfra tests/test_default.py --connection=docker'

  STDOUT:


  STDERR:
```
With ansible version 2.2.1 everything works fine.

--- a/testinfra/test/test_modules.py
+++ b/testinfra/test/test_modules.py
@@ -312,7 +312,7 @@ def test_ansible_module(TestinfraBackend, Ansible):
     with pytest.raises(Ansible.AnsibleException) as excinfo:
         Ansible("file", "path=/etc/passwd an_unexpected=variable")
     tb = str(excinfo.value)
-    assert 'unsupported parameter for module: an_unexpected' in tb
+    assert 'unsupported parameter' in tb.lower()
 
     with pytest.raises(Ansible.AnsibleException) as excinfo:
         Ansible("command", "zzz")

--- a/testinfra/utils/ansible_runner.py
+++ b/testinfra/utils/ansible_runner.py
@@ -31,7 +31,7 @@ if _ansible_major_version == 1:
     import ansible.runner
     import ansible.utils
 elif _ansible_major_version == 2:
-    import ansible.cli
+    import ansible.cli.playbook
     import ansible.executor.task_queue_manager
     import ansible.inventory
     import ansible.parsing.dataloader
@@ -138,8 +138,8 @@ class AnsibleRunnerV2(AnsibleRunnerBase):
         super(AnsibleRunnerV2, self).__init__(host_list)
         _reload_constants()
         self.variable_manager = ansible.vars.VariableManager()
-        self.cli = ansible.cli.CLI(None)
-        self.cli.options = ansible.cli.CLI(None).base_parser(
+        self.cli = ansible.cli.playbook.PlaybookCLI(None)
+        self.cli.options = self.cli.base_parser(
             connect_opts=True,
             meta_opts=True,
             runas_opts=True,
@@ -155,7 +155,7 @@ class AnsibleRunnerV2(AnsibleRunnerBase):
         self.cli.options.connection = "smart"
         self.loader = ansible.parsing.dataloader.DataLoader()
         if self.cli.options.vault_password_file:
-            vault_pass = ansible.cli.CLI.read_vault_password_file(
+            vault_pass = self.cli.read_vault_password_file(
                 self.cli.options.vault_password_file, loader=self.loader)
             self.loader.set_vault_password(vault_pass)
 

